ui/app/hooks/useCancelTransaction.js

```diff
@@ -20,7 +20,10 @@ import { getConversionRate, getSelectedAccount } from '../selectors'
  */
 export function useCancelTransaction(transactionGroup) {
   const { primaryTransaction, initialTransaction } = transactionGroup
-  const gasPrice = primaryTransaction.txParams?.gasPrice
+  const gasPrice =
+    parseInt(primaryTransaction.txParams?.gasPrice, 16) < 0
+      ? '0x0'
+      : primaryTransaction.txParams?.gasPrice
```

Comment:
`parseInt` is not safe for Ethereum quantities.
```suggestion
  const gasPrice =
    primaryTransaction.txParams?.gasPrice?.startsWith('-')
      ? '0x0'
      : primaryTransaction.txParams?.gasPrice
```